app/components/events/view/overview/event-apps.js

```diff
@@ -1,7 +1,17 @@
 import classic from 'ember-classic-decorator';
 import { classNames } from '@ember-decorators/component';
 import Component from '@ember/component';
+import { computed } from '@ember/object';
+import ENV from 'open-event-frontend/config/environment';
 
 @classic
 @classNames('ui', 'fluid', 'card')
-export default class EventApps extends Component {}
+export default class EventApps extends Component {
+    @computed('eventId')
+  get webAppGeneratorUrl() {
+    if (this.authManager.currentUser) {
```

Comment:
No need for this check

Reply:
@iamareebjamal, otherwise one test will fail in travis ci as it doesn't sign in to test this feature. I included this only after I found that failing case.
![image](https://user-images.githubusercontent.com/51796498/98502968-712b9180-2279-11eb-96fa-884a40cf5654.png)
Travis CI build - https://travis-ci.com/github/fossasia/open-event-frontend/builds/199132180

Reply:
You need to set the email of the user in the test. Not checking the case is even worse